orion-server/src/main/java/com/pinterest/orion/core/automation/sensor/kafka/KafkaSensor.java

```diff
@@ -48,30 +51,38 @@ public final void observe(Cluster cluster) throws Exception {
 
   public abstract void sense(KafkaCluster cluster) throws Exception;
 
+  private static Map<String, Object> getClusterConfMap(Cluster cluster) {
+    Map<String, Object> confMap = new HashMap<>();
+    if (cluster.containsAttribute(Cluster.ATTR_CONF_KEY)) {
+      confMap = cluster.getAttribute(Cluster.ATTR_CONF_KEY).getValue();
+    }
+    return confMap;
+  }
+
   protected static boolean containsKafkaAdminClientClusterRequestTimeoutMilliseconds(Cluster cluster) {
-    return cluster.containsAttribute(ATTR_KAFKA_ADMIN_CLIENT_CLUSTER_REQUEST_TIMEOUT_MILLISECONDS_KEY);
+    return getClusterConfMap(cluster).containsKey(ATTR_KAFKA_ADMIN_CLIENT_CLUSTER_REQUEST_TIMEOUT_MILLISECONDS_KEY);
```

Comment:
Is `getClusterConfMap(cluster)` confirmed to be nun-null?

Reply:
This is what I have 

```
  private static Map<String, Object> getClusterConfMap(Cluster cluster) {
    Map<String, Object> confMap = new HashMap<>();
    if (cluster.containsAttribute(Cluster.ATTR_CONF_KEY)) {
      confMap = cluster.getAttribute(Cluster.ATTR_CONF_KEY).getValue();
    }
    return confMap;
  }
```

By theory, if cluster has this attribute, it should not be empty. I will add one extra layer to make sure it wont get null value.